client-go/cmd/apps.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/apps"
+	"github.com/deis/deis/client-go/controller/models/config"
+)
+
+// AppCreate creates an app with the application
+func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
+	c, err := client.Load()
+
+	fmt.Print("Creating Application... ")
+	quit := progress()
+	app, err := apps.Create(*c, id)
+
+	quit <- true
+	<-quit
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("done, created %s\n", app.ID)
+
+	if buildpack != "" {
+		configValues := map[string]string{
+			"BUILDPACK_URL": buildpack,
+		}
+		err = config.Set(*c, app.ID, configValues)
```

Comment:
> if the `if` line would then be too long

Of course, although our [coding standards](http://docs.deis.io/en/latest/contributing/standards/#code-standards) allow lines up to 99 chars, so that shouldn't happen often.